app/components/find/courses/apply_component/view.rb

```diff
@@ -4,13 +4,15 @@ module Find
   module Courses
     module ApplyComponent
       class View < ViewComponent::Base
-        attr_reader :course
+        attr_reader :course, :preview, :utm_content
 
         delegate :application_status_open?, :provider, to: :course
 
-        def initialize(course)
+        def initialize(course, preview, utm_content: nil)
```

Comment:
Can we make this a keyword argument? This way, we can write `View.new(course, utm_content: "something")` for non-previews and `View.new(course, preview: true, utm_content: "something")`. Boolean positional arguments are difficult to guess what they do at a glance.

Additionally, we shouldn't need to guess :)